src/main/scala/cromwell/engine/backend/BackendCall.scala

```diff
@@ -175,7 +175,10 @@ trait BackendCall {
       else
         Future.successful(Option(dockerImage))
 
-    call.task.runtimeAttributes.docker map hashDockerImage getOrElse Future.successful(None) map hashGivenDockerHash
+    if (workflowDescriptor.configCallCaching)
```

Comment:
I remember now... the `Hashing` object currently does not have a `WorkflowDescriptor` reference so it can't tell if caching is enabled.  However, since I basically am doing `if (caching_is_enabled) Option(calculate_hash) else None` in a bunch of places, maybe it's worth passing `WorkflowDescriptor` so it can return an `Option[SymbolHash]`